mne_bids/write.py

```diff
@@ -1015,17 +1015,20 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
         raise ValueError('`t1w` must be a path to a T1 weighted MRI data file '
                          ', or a nibabel image object, but it is of type '
                          '"{}"'.format(type(t1w)))
-    t1_mgh = nib.MGHImage(t1w.dataobj, t1w.affine)
 
+    t1w = nib.Nifti1Image(t1w.dataobj, t1w.affine)
+    # XYZT_UNITS = NIFT_UNITS_MM (10 in binary or 2 in decimal)
+    # seems to be the default for Nifti files
+    t1w.header['xyzt_units'] = np.array(10, dtype='uint8')
```

Comment:
where did you get the information from? Perhaps we can have a short link in the comments? 

Also: Are you potentially overwriting values in `.header['xyzt_units']`?

Reply:
here: https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1fields/nifti1fields_pages/xyzt_units.html. Just need a bit of googling, not sure if it's necessary. I feel too many comments break readability